recipes/pytest-json-report/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "pytest-json-report" %}
+{% set version = "1.2.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 628b2a3a8f6edf069bf502ad8f65694ce6f2e4f86e73db442f18122389dafc38
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - pytest >=4.2.0
+    - pytest-metadata
+    - python
+
+test:
+  imports:
+    - pytest_jsonreport
+
+about:
+  home: https://pypi.org/project/pytest-json-report/
```

Comment:
This should probably be https://github.com/numirias/pytest-json-report, but you can fix it in the feedstock.